lua/import/pickers/snacks.lua

```diff
@@ -1,20 +1,34 @@
 local constants = require("import.core.constants")
 
-local ok, pick = pcall(require, "snacks.picker")
+local ok, snacks = pcall(require, "snacks")
 if not ok then
   return function()
     error("snacks.nvim not found. Please install it to use this picker.", 0)
   end
 end
 
+local user_layout = vim.tbl_get(snacks, "config", "picker", "layout", "layout") or {}
+
+local overrides = {
+  width = constants.width,
+  height = constants.height,
+  min_height = constants.min_height,
+  min_width = constants.min_width,
+}
+
+local merged_layout = vim.tbl_deep_extend("force", user_layout, overrides)
+
+local layout = { layout = merged_layout, preview = false }
+
```

Comment:
In my configuration `layout` is a string, but it can also be a table or even a function.
You can use snacks picker API to retrieve the layout from configuration:

```suggestion
local user_layout = snacks.picker.config.layout(snacks.picker.config.get())

local overrides = {
  layout = {
    width = constants.width,
    height = constants.height,
    min_height = constants.min_height,
    min_width = constants.min_width,
  },
  preview = false,
}

local layout = vim.tbl_deep_extend("force", user_layout, overrides)

```

I tested the changes using my default layout and it correctly overrides the width and height while removing the preview.
It also preserves other configurations, such as `win.input/list.keys`.

Reply:
Nice! I'm going to go ahead and merge 👌